### PR TITLE
FAQ update

### PR DESCRIPTION
**Pull request by ashleewang888**
- layout modified
- content updated

### CODE REVIEW

File: versioned_docs/version-1.6(WIP)/faq.md [hunk 1]
```diff
@@ -16,46 +16,61 @@ In short, a [custodian cell](depAndWthd.md#custodian-cell) is an asset held in t
 
 ----
 
+### How to get Layer 1 Block Number?
+
+Defi contracts often require interests to be calculated by block number. However, both v0 and v1 networks do not have a fixed layer2 block interval (it is affected by layer1 block interval).
```
Comment (Flouse) [inline versioned_docs/version-1.6(WIP)/faq.md:21]:
```suggestion
DeFi contracts often require interests to be calculated by block number. However, both v0 and v1 networks do not have a fixed layer2 block interval (it is affected by layer1 block interval).
```

File: versioned_docs/version-1.6(WIP)/faq.md [hunk 2]
```diff
@@ -16,46 +16,61 @@ In short, a [custodian cell](depAndWthd.md#custodian-cell) is an asset held in t
...
 
 This constraint is a temporary mechanism and will be removed in a future release. Godwoken is the world's first rollup running on a UTXO blockchain. The answer is not ready-made, and we have to explore different ways to find the best design. It's your feedback and patience that help us improve Godwoken and make it better in future iterations. Thank you.
 
 ----
 
-### How to get Layer 1 Block Number?
+### Failed to transfer token via Metamask on Godwoken
```
Comment (Flouse) [inline versioned_docs/version-1.6(WIP)/faq.md:57]: Support for native token transfer
Previously when using Metamask on Godwoken, it was necessary to add a custom ERC20 contract for pCKB to send pCKB tokens. This requirement is no longer required. pCKB is now treated as a native token on Godwoken and works out of the box with Metamask.

- https://github.com/nervosnetwork/godwoken-polyjuice/pull/173
- https://github.com/nervosnetwork/godwoken/pull/788
- https://github.com/nervosnetwork/godwoken-web3/pull/505

File: versioned_docs/version-1.6(WIP)/faq.md [hunk 3]
```diff
@@ -16,46 +16,61 @@ In short, a [custodian cell](depAndWthd.md#custodian-cell) is an asset held in t
...
+ - mainnet_v1 pCKB_ERC20_Proxy contract: 0x7538C85caE4E4673253fFd2568c1F1b48A71558a (pCKB)
+ - testnet_v1 pCKB_ERC20_Proxy contract: 0xE05d380839f32bC12Fb690aa6FE26B00Bd982613 (pCKB)
+ 
 ----
 
-### CKB transfer on Godwoken testnet via Metamask has failed. What is the solution?
+### Getting the transaction completed faster
 
-In Godwoken, `transaction.to` must be a contract address. In RPCs such as `eth_call`, `eth_estimateGas` and `eth_sendRawTransaction`, the `to` parameter can only be a contract address, not an EOA address.
+Q: The average block confirmation interval on Godwoken is 30s or more, yet I would like to have my transactions completed more quickly, what should I do to make this happen?
```
Comment (Flouse) [inline versioned_docs/version-1.6(WIP)/faq.md:72]: - Avg. Block Time of Godwoken testnet_v1: https://v1.testnet.gwscan.com/
- Avg. Block Time of Godwoken mainnet_v1: https://v1.gwscan.com/

**Review by Flouse — CHANGES_REQUESTED** [review]

**Comment by ashleewang888** [comment]
@Flouse

**Review by Flouse — APPROVED** [review]

**Pull request merged**